zcash_test_vectors/zip_0244.py

```diff
@@ -272,7 +271,7 @@ def hash_type(tx, nHashType, txin):
             SIGHASH_ALL | SIGHASH_ANYONECANPAY,
             SIGHASH_NONE | SIGHASH_ANYONECANPAY,
             SIGHASH_SINGLE | SIGHASH_ANYONECANPAY,
-        ]
+            ]
```

Comment:
avoid changing space if not needed.

Reply:
![image](https://github.com/user-attachments/assets/f28687e8-bcc0-444d-9a9c-5b5454345764)
Just like below, this doesn't show locally as a change. Will check it.

Reply:
Fixed that, not sure why the space had changed, but it shows as unchanged both on github and locally now